### PR TITLE
Post jembi optout

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: go-app-chw.js [hunk 1]
```diff
@@ -436,9 +440,23 @@ go.utils = {
         return formatter();
     },
 
+    get_nurse_id: function(contact) {
```
Comment (imsickofmaps) [inline go-app-chw.js:443]: Is this not the same as the patient ID function?

File: go-app-chw.js [hunk 2]
```diff
@@ -485,6 +519,28 @@ go.utils = {
...
+            "id": go.utils.get_nurse_id(contact),
+            "optoutreason": go.utils.get_nurse_optoutreason(contact)
+        };
+        return JSON_template;
+    },
+
+    jembi_nurse_optout_send_json: function(contact, user, type, im, metric_prefix) {
+        var built_json = go.utils.build_nurse_jembi_json(im, contact, user, type);
+        return go.utils
+            .jembi_json_api_call('post', null, built_json, 'optout', im);
```
Comment (imsickofmaps) [inline go-app-chw.js]: Double-check this on response from Pierre that optout endpoint not diff for nurses

**Comment by imsickofmaps** [comment]
Just a confirmation on DRYness and URL then :+1:

File: go-app-chw.js (hunk 1 above)
Comment (gsvr) [inline go-app-chw.js:443]: It's close, but the extra names differ `nc_` and there is no 'none' option for nurses.

**Pull request merged**